src/config/graphical-session/xorg.md

```diff
@@ -106,6 +106,16 @@ be used to start X sessions from the console. For example, to use i3, edit
 exec /bin/i3
 ```
 
+To start arbitrary programs together with an X session, add them in `~/.xinitrc`
+before the last line. For example, to start
```

Comment:
` <...> before starting the window manager / DE.`
Makes the example code block is superfluous in my opinion (or could be integrated in the above code block even)

Reply:
True. I've tweaked this a bit.